lib/Model/Listing/AbstractListing.php

```diff
@@ -445,4 +447,65 @@ public function getConditionVariablesFromSetCondition()
     {
         return $this->conditionVariablesFromSetCondition;
     }
+
+    public function getData(): ?array
```

Comment:
BC break, don't set the return value, what if someone else extends from AbstractListing?

Reply:
Ok, changed this as it is not really necessary.

Reply:
But as this function did not exist before this PR I actually do not see a BC break.

Reply:
but in many other Listings it did

Reply:
I only found it in https://github.com/pimcore/pimcore/blob/bb4fd386fd09d3e5f4e26041f7aef2eccae0aa7e/models/Notification/Listing.php#L52

But anyway: The return type hint does not add any value imho, so I am fine as it is now (without return type hint)